modules/api/pkg/handler/v1/project/project.go

```diff
@@ -259,7 +259,7 @@ func ListEndpoint(userInfoGetter provider.UserInfoGetter, projectProvider provid
 			return nil, common.KubernetesErrorToHTTPError(err)
 		}
 
-		if req.DisplayAll && userInfo.IsAdmin {
+		if (req.DisplayAll && userInfo.IsAdmin) || (userInfo.IsGlobalViewer) {
```

Comment:
```suggestion
		if (req.DisplayAll && (userInfo.IsAdmin || userInfo.IsGlobalViewer) {
```

Shouldn't it be something like this?